### ISSUE TITLE
Ability to clear date

### ISSUE THREAD

**Issue by kevintraver** [quantosobra/ember-semantic-ui-calendar#1]
Could be something like this:

![image](https://cloud.githubusercontent.com/assets/196406/26326913/94b501ba-3efa-11e7-8147-1563309843f3.png)

I know that semantic-ui-calendar has the ability to clear the date

 `this.$().calendar('clear');`

so it would be nice if it could integrate with Ember actions

**Comment by eduardoweiland** [comment]
That seems good. Do you think that an `allowClear` option (and possibly a `clearIcon` too) would be enough? Or do you think it could be more customizable?

**Comment by kevintraver** [comment]
I think `allowClear` and `clearIcon` is good enough to start with.

**Comment by eduardoweiland** [comment]
This seems a little more complicated that I thought at first. The [input element in Semantic UI](https://semantic-ui.com/elements/input.html#icon) does not support two icons in the same input, so this would require custom styling.

**Comment by kevintraver** [comment]
Yeah, that makes it difficult. Maybe something like this.

```hbs
<div class="ui left right icon input">
  <i class="left users icon"></i>
  <input type="text" placeholder="Search users...">
  <i class="right remove icon"></i>
</div>
```
```css
.ui[class*="left right icon"].input > i.left.icon {
  right: inherit;
}

.ui[class*="left right icon"].input > input {
  padding-left: 2.67142857em !important;
  padding-right: 2.67142857em !important;
}
```

Or maybe don't worry about styling, and let the user take care of it.

**Comment by eduardoweiland** [comment]
@kevintraver Can you test the PR #2 and see if it is what you're expecting?

**Comment by kevintraver** [comment]
Awesome! Looks great!

**Comment by eduardoweiland** [comment]
Released in v0.1.0
